backend/dataall/core/permissions/permissions.py

```diff
@@ -27,7 +27,6 @@
 MANAGE_GLOSSARIES = 'MANAGE_GLOSSARIES'
```

Comment:
@nikpodsh why didn't we include the glossaries permissions under the Catalog module?

Reply:
We simply forgot 😨

Reply:
Should we open a new PR for that?